devices/fields/tim/v2/tim2_uif.yaml

```diff
@@ -0,0 +1,15 @@
+_include:
+  - ../basic.yaml
+  - ../tim_gp1.yaml
+  - ../generic.yaml
+  - ../tim_ckd.yaml
+  - ../ccr.yaml
+  - ../v1/occe.yaml
+  - ccm_extended.yaml
+
+CNT:
+  CNT: [0, 0x7FFFFFFF]
+ARR:
+  ARR: [0, 0xFFFFFFFF]
```

Comment:
I might be missing something but according to [RM0529](https://www.st.com/resource/en/reference_manual/rm0529-stm32wb05xz-ultralow-power-wireless-32bit-mcu-armbased-cortexm0-with-bluetooth-low-energy-and-24-ghz-radio-solution-stmicroelectronics.pdf)

![image](https://github.com/user-attachments/assets/54085bf7-ad00-412b-9d47-9217589fe258)

Reply:
Oh. Sorry. It is my fault.

Reply:
Should be fixed now.